src/composables/ViewRoutingSlip/usePaymentInformation.ts

```diff
@@ -26,6 +29,22 @@ export default function usePaymentInformation (_, context) {
     return payments && payments[0].paymentMethod === PaymentMethods.CHEQUE
   })
 
+  function adjustRoutingSlipChequeNumber (num: string, i: number = 0) {
```

Comment:
What does i represent? I'm guessing index of some sort? Index of what?

`paymentIndex` might be a bit more appropriate

Reply:
it's paymentIndex, just changed